pkg/ebpf/c/tracer-ebpf.c

```diff
@@ -638,6 +660,29 @@ int kprobe__udp_sendmsg(struct pt_regs* ctx) {
     return 0;
 }
 
+SEC("kprobe/udp_sendmsg/rhel")
+int kprobe__udp_sendmsg__rhel(struct pt_regs* ctx) {
+    struct sock* sk = (struct sock*)PT_REGS_PARM1(ctx);
+    size_t size = (size_t)PT_REGS_PARM4(ctx);
```

Comment:
I might be helpful down the line to call out this line as the only part that changes, since it's not clear how this differs from `kprobe__udp_sendmsg` at a glace (same comment for the tcp analog of this function)